src/components/AddPlaidBankAccount/index.native.js

```diff
@@ -0,0 +1,183 @@
+import _ from 'underscore';
+import React from 'react';
+import {
+    ActivityIndicator,
+    View,
+} from 'react-native';
+import lodashGet from 'lodash/get';
+import {withOnyx} from 'react-native-onyx';
+import Log from '../../libs/Log';
+import PlaidLink from '../PlaidLink/index';
+import {
+    clearPlaidBankAccountsAndToken,
+    fetchPlaidLinkToken,
+    getPlaidBankAccounts,
+    setBankAccountFormValidationErrors,
+    showBankAccountErrorModal,
+} from '../../libs/actions/BankAccounts';
+import ONYXKEYS from '../../ONYXKEYS';
+import styles from '../../styles/styles';
+import themeColors from '../../styles/themes/default';
+import compose from '../../libs/compose';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import ExpensiPicker from '../ExpensiPicker';
+import Text from '../Text';
+import * as ReimbursementAccountUtils from '../../libs/ReimbursementAccountUtils';
+import ReimbursementAccountForm from '../../pages/ReimbursementAccount/ReimbursementAccountForm';
+import getBankIcon from '../Icon/BankIcons';
+import Icon from '../Icon';
+import {propTypes, defaultProps} from './plaidBankPropTypes';
+
+const plaidBankPropTypes = {
+    ...withLocalizePropTypes,
+    ...propTypes,
+};
+
+class AddPlaidBankAccount extends React.Component {
```

Comment:
I know you're probably still working on this - but we should avoid duplicating this code and do something like have a `BaseAddPlaidBankAccount` component and then in `index.js` and `index.native.js` pass whatever is different about the two implementations as props.

Reply:
Got it yes, was going to do this too